Celeste.Mod.mm/Mod/Core/CoreModuleSettings.cs

```diff
@@ -266,6 +267,19 @@ public class CoreModuleSettings : EverestModuleSettings {
             }
         }
 
+        private bool _ColorizedLogging = true;
```

Comment:
Are we OK with this being opt-_out_? 🤔

Reply:
I got referred to a previous conversation: yes we are OK because there's no performance impact